train.py

```diff
@@ -37,6 +40,10 @@ def main():
   """
   args = argparse.parse_cmd(arguments)
   args['server'] = alchemy_api.start_training(args['server'])
+  if args['use_wandb'] and "tensorflow" in args['framework']:
```

Comment:
add a comment in the code to explain why there is a condition on tensorflow

Reply:
Let me know if this comment is okie. or should we be more explicit as to why we don't report Upstride.